reconcile/queries.py

```diff
@@ -547,6 +557,7 @@ def get_aws_accounts(
     sharing=False,
     terraform_state=False,
     ecrs=True,
+    cleanup=False,
```

Comment:
Now there is a new way to use typed query to get aws account instead of adding more params to this function, an example is like this https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/gql_definitions/terraform_tgw_attachments/aws_accounts.gql.

You can create a new one for new integration, and reuse some fragments.

Reply:
Good point Rafa! In this case, keep using the old dict-based queries.

Reply:
added a comment on 47b58c56 around this.